tests/integration_tests/performance/test_rate_limiter.py

```diff
@@ -207,7 +207,10 @@ def _check_rx_rate_limiting(test_microvm):
 
     # Second step: check bandwidth when rate limiting is on.
     print("Run guest RX iperf for rate limiting without burst")
-    _check_rx_bandwidth(test_microvm, eth1.guest_ip, expected_kbps)
+    observed_kbps = _get_rx_bandwidth_with_duration(
+        test_microvm, eth1.guest_ip, IPERF_TRANSMIT_TIME
+    )
+    assert _diff(observed_kbps, expected_kbps) < MAX_BYTES_DIFF_PERCENTAGE
```

Comment:
I don't really get what's the point for this change? Why inline this?

Reply:
I found this whole file hard to understand at first because of all of the "helper" functions, so I inlined this one while I was doing other changes.